tests/migration/online_migration/zypper_migration.pm

```diff
@@ -83,7 +83,8 @@ sub run {
         elsif ($out =~ $zypper_migration_conflict)
         {
             if (check_var("BREAK_DEPS", '1')) {
-                send_key '1';
+                # for leap to sle migration, we need choose 2 to resolve conflicts
```

Comment:
This should be a workaround for issue, normally we do not need resolve any conflict during migration.

Reply:
added comments for this.